### DISCUSSION TITLE
Diablo 1 issue with gameplay

### DISCUSSION THREAD

**Discussion by Mett0ttim** [diasurgical/DevilutionX#7592] [General]
I am playing as a Rogue. I have reached lvl 27 on Nightmare. I have been experiencing something strange over the last few days when playing. I'll describe it: I start a game, I play, a monster drops an item, before picking it up let's say I die. I respawn at my last save point, and go to collect something dropped, but no matter how much I click on the object my character can't pick it up. Another example is I start a game, die, respawn, then cast a town portal spell. No matter what I do I can't enter the portal. There is something going on here over my head. So far, my solution, is to quit out of the program entirely and then restart a game. The problem stops until I die, the it begins again. Any ideas?

**Comment by yuripourre** [comment]
@Mett0ttim can you provide a save game with this issue? It would be very helpful.

**Comment by AJenbo** [comment]
> I start a game

There is a lot of information missing here

**Comment by qndel** [comment]
"let's say I die. I respawn at my last save point" - are you sure you are playing diablo 1? lol
